morphomuseum.csl

```diff
@@ -0,0 +1,253 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="en-US">
+  <info>
+    <title>MorphoMuseuM</title>
```

Comment:
Since http://morphomuseum.com/ mentions that the journal is also known as "M3", you might want to add another line here, i.e.:

```
<title>MorphoMuseuM</title>
<title-short>M3</title-short>
```